src/gui/entry/EntryModel.cpp

```diff
@@ -127,9 +131,29 @@ QVariant EntryModel::data(const QModelIndex& index, int role) const
         return QVariant();
     }
 
+    /**
+     * @author Fonic <https://github.com/fonic>
+     * Copied date/time formatting string from 'EditWidgetProperties.cpp'
+     * TODO: this shouldn't be hardcoded, neither here nor in 'EditWidgetProperties.cpp',
+     *       instead this should be global and user-configurable (or derived automatically
+     *       from current regional settings)
+     */
+    QString timeFormat("d MMM yyyy HH:mm:ss");
```

Comment:
We could do that, but I think this should be a global, user-configurable setting as the way timestamps are displayed varies from one country/language to another.

How about putting it in _src/core/Config.cpp_:
```c++
void Config::init(const QString& fileName)
{
    m_defaults.insert("DateTimeFormat", "d MMM yyyy HH:mm:ss");
}
```
and then fetching it from the config whenever it's needed?